03_analysis/match_minima.py

```diff
@@ -94,28 +95,33 @@ def plot_violin_signed(msds, ff_list, what_for='talk'):
         "paper" or "talk"
 
     """
+    
     # print stats on number of datapoints in each method
     temp = msds.T
-    for l in temp:
+    for l, m in zip(temp, ff_list):
         print(f"Total number of unique molecules: {l.shape}")
-        print(f"Count of non-nan molecules: {np.count_nonzero(~np.isnan(l))}")
+        print(f"Count of non-nan molecules for {m}: {np.count_nonzero(~np.isnan(l))}")
 
     # create dataframe from list of lists
     df = pd.DataFrame.from_records(msds, columns=ff_list)
+    #change order of columns and order of plotting
+    df = df.loc[:, ['GAFF', 'GAFF2', 'MMFF94', 'MMFF94S', 'OPLS3e', 'Smirnoff99Frosst', 'OpenFF-1.0', 'OpenFF-1.1', 'OpenFF-1.2']]
```

Comment:
I thought this script should be able to read in the labels from the input file so they wouldn't have to be encoded into the script. Hm, though your comment suggests that this is a temporary fix for change of order. Maybe let's not commit directly into master. But we should keep this version of the code in case we need to edit figures later. @davidlmobley - any input on how to best handle this?

Reply:
Yes, it was a temporary fix to change the order of FFs to avoid having to run the whole analysis again. Resolved in b59b5b5